docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM continuumio/anaconda3:4.1.1
+MAINTAINER "Greg Way" <gregway@mail.med.upenn.edu>
+
+# Install base packages and R
+RUN apt-get update --fix-missing && apt-get install -y \
```

Comment:
Can you just install R from the R conda channel? This way you can specify the exact version? May make your life easier... that is if there isn't some bug with using checkpoint and R for conda.